changelogs/feature/disable-trace-logging-by-config.json

```diff
@@ -0,0 +1,9 @@
+{
+"author": "smirglagit",
+
+"pullrequestId": -1,
+
+"message": "Added new configuration parameter to handle tracing options",
+
+"issue": "704"
```

Comment:
Issue applies to github issue, which in this case does not exist.
Please remove it